akka-docs/rst/java/http/server-side/websocket-support.rst

```diff
@@ -122,5 +122,5 @@ The handling code itself will be the same as with using the low-level API.
 
 See the `full routing example`_.
 
-.. _example: @github@/akka-docs-dev/rst/java/code/docs/http/javadsl/server/WebSocketCoreExample.java
-.. _full routing example: @github@/akka-docs-dev/rst/java/code/docs/http/javadsl/server/WebSocketRoutingExample.java
\ No newline at end of file
+.. _example: @github@/akka-docs/rst/java/code/docs/http/javadsl/server/WebSocketRoutingExample.java
```

Comment:
Full routing is below, and that is fixed correctly, but the above one is the "example" link and should not point to the same sample code. So, just fix the example link to point to `WebSocketCoreExample.java` and this can be merged, thanks!